src/Utils/MODES.java

```diff
@@ -1,48 +1,45 @@
 package Utils;
 
 public class MODES {
-    public static final String SELECT = "Select";
-    public static final String ASSOCIATION_LINE = "AssociationLine";
-    public static final String GENERALIZATION_LINE = "GeneralizationLine";
-    public static final String COMPOSITION_LINE = "CompositionLine";
-    public static final String CLASS = "Class";
-    public static final String USE_CASE = "Use Case";
+    public static String SELECT = "Select";
+    public static String ASSOCIATION_LINE = "AssociationLine";
+    public static String GENERALIZATION_LINE = "GeneralizationLine";
+    public static String COMPOSITION_LINE = "CompositionLine";
+    public static String CLASS = "Class";
+    public static String USE_CASE = "Use Case";
 
     public static String getModeButtonName(String mode) {
-        switch (mode) {
-            case SELECT:
-                return "Select";
-            case ASSOCIATION_LINE:
-                return "<html>Association<br>Line</html>";
-            case GENERALIZATION_LINE:
-                return "<html>Generalization<br>Line</html>";
-            case COMPOSITION_LINE:
-                return "<html>Composition<br>Line</html>";
-            case CLASS:
-                return "Class";
-            case USE_CASE:
-                return "Use Case";
-            default:
-                return "";
+        if (mode.equals(SELECT)) {
+            return "Select";
+        } else if (mode.equals(ASSOCIATION_LINE)) {
+            return "<html>Association<br>Line</html>";
+        } else if (mode.equals(GENERALIZATION_LINE)) {
+            return "<html>Generalization<br>Line</html>";
+        } else if (mode.equals(COMPOSITION_LINE)) {
+            return "<html>Composition<br>Line</html>";
+        } else if (mode.equals(CLASS)) {
+            return "Class";
+        } else if (mode.equals(USE_CASE)) {
+            return "Use Case";
+        } else {
+            return "";
         }
     }
-
     public static String getModeImagePath(String mode) {
-        switch (mode) {
-            case SELECT:
-                return "imgs/select.png";
-            case ASSOCIATION_LINE:
-                return "imgs/association-line.png";
-            case GENERALIZATION_LINE:
-                return "imgs/generation-line.png";
-            case COMPOSITION_LINE:
-                return "imgs/composition-line.png";
-            case CLASS:
-                return "imgs/class.png";
-            case USE_CASE:
-                return "imgs/use-case.png";
-            default:
-                return "";
+        if(mode.equals(mode)){
```

Comment:
Hi @alexeyleping

In the following code snippet, there is a bug where `mode` is incorrectly compared to itself using the `equals()` method. This results in the condition always being true, causing only the first `if` statement to be accessible.

https://github.com/1chooo/uml-editor/blob/f4621b4232c97dcd59709a339388250d948b174c/src/Utils/MODES.java#L29-L30

```java
if(mode.equals(mode)){ 
    return "imgs/select.png"; 
}
```

**Bug Fix:**

To fix this issue, we need to compare `mode` to the constant `SELECT` instead of itself. The corrected code should be:

```java
if(mode.equals(SELECT)){ 
    return "imgs/select.png"; 
}
```

This change ensures that the condition correctly checks if `mode` is equal to `SELECT`, allowing the subsequent `if` statements to be evaluated properly.